src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -82,6 +82,8 @@ public static LineWrappingOptions ofBoolean(boolean val) {
         TokenTypes.RCURLY,
         TokenTypes.LITERAL_NEW,
         TokenTypes.ARRAY_INIT,
+        TokenTypes.LITERAL_DEFAULT,
```

Comment:
Please add corresponding `@see CaseHandler#...` tag to javadoc above.

Reply:
Added `* @see CaseHandler#getIndentImpl()`

Reply:
Resolved.